### DISCUSSION TITLE
Selenium 4.3.0 support

### DISCUSSION THREAD

**Discussion by OldEngineer1911** [appium/java-client#1726] [General]
Hi all,

is there some plan for bump selenium dependency to 4.3.0?

Thanks!

**Comment by valfirst** [comment]
@OldEngineer1911 I think Appium Java Client `8.1.1` is fully compatible with Selenium `4.3.0`. The minimum version is `4.2.1`. In case some Selenium `4.3.0` features are needed, the dependency of the required version can be added.

**Reply by OldEngineer1911** [reply]
Sorry, the question was not clear. I received:
![Screenshot 2022-07-26 at 21 33 46](https://user-images.githubusercontent.com/43892318/181096632-4ad4cb3c-0f18-4f90-af9b-a89362abcb67.png)

Does not cause any issues now, so I am just asking :)

**Reply by valfirst** [reply]
This is a dependency conflict (not sure which build you use, if use any), depending on your requirements `4.3.0` or `4.2.1` version can be chosen, Appium Java client `8.1.1` should work with any of these versions.

**Reply by OldEngineer1911** [reply]
Yes, it's a maven issue. Ok, if it should work with 4.3.0 the issue is probably somewhere else.
